nototools/noto_lint.py

```diff
@@ -736,6 +603,47 @@ def _check_unused_names():
           warn('name/unused', 'Name', 'Name table has record #%d: "%s"' %
                (i, names[i]), is_error=False)
 
+    def _noto_font_from_font_props(font_props):
+      fields = """
+          filepath,family,style,script,variant,width,weight,slope,fmt,
+          manufacturer,license_type,is_hinted,is_mono,is_UI,is_display,
+          is_cjk,subset
```

Comment:
I'm planning on nuking the FontProps definition and just using NotoFont.  Might rename NotoFont to FontProps, but either way I don't think I need two definitions of nearly the same struct.